nipype/interfaces/cmtk/nx.py

```diff
@@ -167,7 +167,7 @@ def average_networks(in_files, ntwk_res_file, group_id):
             data = ntwk.edge[edge[0]][edge[1]]
             if ntwk.edge[edge[0]][edge[1]]["count"] >= count_to_keep_edge:
                 for key in list(data.keys()):
```

Comment:
Curious why this line wasn't simplified to `list(data)`...

```suggestion
                for key in data:
```

Since we're not removing keys in the list body, it is not necessary to copy the keys into a list for safe iteration.

Reply:
Sometimes I might need to re-run ruff again with rules already applied. I apply each rule sequentially.